torch/_functorch/_aot_autograd/autograd_cache.py

```diff
@@ -390,9 +403,17 @@ def load(self, example_inputs, fx_config: _CompileFxKwargs) -> CompiledFxGraph:
             },
             payload_fn=lambda: json.dumps(cache_info),
         )
+        self.example_inputs = example_inputs
```

Comment:
why do we need to do this ?

Reply:
It makes it so we can pass it to the post_compile() below. The goal here is to split the post_compile from the cache loading, so that we don't set the global constants until *after* we're sure the cache successfully hit on both